database/migrations/2016_02_11_155642_add_extras_order_on_user_profile_customization.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ *    Copyright 2015 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed in the hopes of
+ *    attracting more community contributions to the core ecosystem of osu!
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+use Illuminate\Database\Migrations\Migration;
+use App\Models\UserProfileCustomization;
+
+class AddExtrasOrderOnUserProfileCustomization extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('user_profile_customizations', function ($table) {
+            $table->string('extras_order')->default(serialize(UserProfileCustomization::$sections));
```

Comment:
actually, keep the default empty so user will always have default ordering until they explicitly set the order.